demos/human_pose_estimation_demo/src/send_human_pose.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <utility>
+#include <vector>
+
+#include <samples/common.hpp>
+
+#include "human_pose_estimator.hpp"
+#include "send_human_pose.hpp"
+
+
+namespace human_pose_estimation {
+
+char POSE_COCO_BODY_PARTS[][18] = { "Nose", "Neck", "RShoulder", "RElbow", "RWrist",  "LShoulder",
+                                   "LElbow", "LWrist", "RHip", "RKnee", "RAnkle", "LHip",
+                                    "LKnee", "LAnkle", "REye", "LEye", "REar", "LEar", "Bkg"};
+
+void sendHumanPose(const std::vector<HumanPose>& poses) {
+
+    if (!poses.empty()) {
+        std::time_t result = std::time(nullptr);
+        std::cout << std::asctime(std::localtime(&result)) << std::endl;
+     }
+
```

Comment:
Why the format of printing the results is changed?

Reply:
I wanted to put it apart, cause it would be interesting if people can send joints over other applications.

Reply:
Some internal tests could depend on that behavior. I will ask about it.

Reply:
It seems we can keep it.